tests/ts_simple_select.erl

```diff
@@ -27,14 +27,18 @@
 -export([confirm/0]).
 
 confirm() ->
-    TestType = normal,
-    DDL = ts_util:get_ddl(),
-    Data = ts_util:get_valid_select_data(),
-    Qry = ts_util:get_valid_qry(),
+    Table = ts_data:get_default_bucket(),
+    DDL = ts_data:get_ddl(),
+    Data = ts_data:get_valid_select_data(),
+    Qry = ts_data:get_valid_qry(),
     Expected =
-        {ok, {ts_util:get_cols(small),
-         ts_util:exclusive_result_from_data(Data, 2, 9)}},
-    Got = ts_util:ts_query(
-            ts_util:cluster_and_connect(single), TestType, DDL, Data, Qry),
+        {ok, {ts_data:get_cols(small),
+         ts_data:exclusive_result_from_data(Data, 2, 9)}},
+
+    Cluster = ts_setup:start_cluster(1),
+    ts_setup:create_bucket_type(Cluster, DDL, Table),
+    ts_setup:activate_bucket_type(Cluster, Table),
+    ts_ops:put(Cluster, Table, Data),
+    Got = ts_ops:query(Cluster, Qry),
     ?assertEqual(Expected, Got),
```

Comment:
I readily admit that this is more granular but almost as opaque as the current testing library.

I'm fine with a preference for a better testing interface, but I'd like to see a concrete example so we can evaluate how much effort it would be to create and to convert the tests to use it.